apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/TableOperationTask.java

```diff
@@ -13,6 +16,7 @@
 @Slf4j
 @Getter
 public abstract class TableOperationTask extends OperationTask<TableMetadata> {
+  private static final String MAINTENANCE_PROPERTY_PREFIX = "maintenance.";
```

Comment:
So this should override the global maintenance config.

Reply:
Yes, in the services/jobs/src/main/java/com/linkedin/openhouse/jobs/services/JobsRegistry.java, incoming props will be override existing one in the jobs.yaml

Reply:
Are these like reserved properties? for eg: like openhouse. prefix that not everyone can update?

Reply:
These are not reserved, they are supposed to be set directly, as opposed to table type, policies, etc